src/jdk.javadoc/share/classes/jdk/javadoc/doclet/Reporter.java

```diff
@@ -94,6 +94,12 @@
      * The positions are all 0-based character offsets from the beginning of content of the file.
      * The positions should satisfy the relation {@code start <= pos <= end}.
      *
+     * @implNote
+     * This implementation throws {@code UnsupportedOperationException}.
+     * The implementation provided by the {@code javadoc} tool to
+     * {@link Doclet#init(Locale, Reporter) initialize} a doclet
+     * overrides this implementation.
```

Comment:
Overrides how?

Reply:
...per the documented spec.   I tried writing `The implementation .... always generates an appropriate diagnostic` but that's not true if an exception is thrown, and writing `The implementation ... does not throw UOE` seems weird.

Reply:
The difficulty you have with ironing out this wording wrinkle suggests that the behavior of the `Reporter` provided to `Doclet.init` should be described in `Doclet.init` rather than here. The same goes for other default methods of Reporter.